### PR TITLE
Typos, clear corrections and minor improvements.

### PR DESCRIPTION
**Pull request by fioddor**
* layout: This file would definitively benefit from **links** to "First contributions" and "Review process" documents, as these are its leading and trailing documents in the logical reading sequence. The latter is hidden/buried in the "Branch policy" and deserves a more prominent display.
* visuals: The **remark** to the author should be marked in italic for consistency. 
* typo: In "Sending the pull requests for review with any prior communication" "with" should be "with**out**".
* typo: **Align** branch name with text. Currently text says `main` and picture displays `master`.
* msg: Instead of "Go to GitHub. Create the Pull Request" I suggest "Go to **your fork** in GitHub and request the (assigned) upstream TC's to pull your changes". Users new to git/github/gitlab use to have a fuzzy understanding of what a pull/merge request is and believe it is some special kind of branch.
* msg: The possibilities explained in "could be useful for code repositories" don't apply to this project due to **size**, not because they are code-specific. In big non-code projects like critical documentation ones they could apply.
* typo: "git flow" -> "**Gitflow**"
* msg: https://www.atlassian.com/git/tutorials/comparing-workflows might be a good external **reference**.

### CODE REVIEW

**Comment by fioddor** [comment]
Once in main, another PR is needed to send the changes upstream (and thereby test and showcase the fork-based workflow).

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -106,7 +110,7 @@ The main ideas behind the branching strategy explained here are:
   * Users are expected to use the latest version released.
   * Anyone should be able to contribute to any project. That means the branching policy should allow anyone to read, but obviously not to write. Assuming a code review process, that means contributors need a branch of their own to make the changes and, from there, open a pull request to the corresponding upstream branch.
 
-By following these ideas, the project `main` branch is the place to go to get the latest changes. Some of them could be still under testing processes. This means `main` is as stable as the testing performed on each Pull Request before merging it into master. It is granted that those changes are part of the roadmap (in other words, may not be fully tested but they are not experimental).
+By following these ideas, the project `master` branch is the place to go to get the latest changes. Some of them could be still under testing processes. This means `master` is as stable as the testing performed on each Pull Request before merging it into master. It is granted that those changes are part of the roadmap (in other words, may not be fully tested but they are not experimental).
```
Comment (alpgarcia) [inline CONTRIBUTING.md:113]: GitHub now uses `main` by default and so does this project.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -133,13 +137,13 @@ In case several versions of the project have to co-exist in production at the sa
...
-  * **Corner case**: if this were a code project and a bug broke an older version with not any possibility of updating from that version to a newer one for a representative number of users, a branch would be created from that point, the fix would be applied on that branch and then merged into the `main` branch. The branch created for the fix is expected to be deleted as soon as users of that branch are able to update to the latest release.
-  * **Experimental development**: sometimes it is needed to break everything to explore new options for future releases (always under the roadmap of the project) while some developments for the next one are still being tested and pending to be merged. In those cases, a new branch will be created until `main` branch gets to a specific release point. At that point the new branch will be reintegrated into `main`.
+  * In case of a bug, it will be fixed on the current `master` branch and not backported. **Users are expected to be always in the latest release**, supporting several versions at a time is currently out of the current possibilities of the project in terms of effort.
```
Comment (alpgarcia) [inline CONTRIBUTING.md:144]: Same as above.

**Review by alpgarcia — CHANGES_REQUESTED** [review]
The only change I requested is to keep using `main`. It is better to update the picture (we can open an issue to not forget) as the actual branch name is `main`.

**Review by alpgarcia — APPROVED** [review]
LGTM, thanks!

**Pull request closed without merging**